packages/client/src/MetricsPublisher.ts

```diff
@@ -71,8 +73,9 @@ export class MetricsPublisher {
         const ensureStarted = pOnce(async () => {
             const node = await this.node.getNode()
             const metricsContext = node.getMetricsContext()
-            const partitionKey = getEthereumAddressFromNodeId(node.getNodeId()).toLowerCase()
+            const partitionKey = (await authentication.getAddress()).toLowerCase()
```

Comment:
```suggestion
            const partitionKey = await authentication.getAddress()
```

`Authentication` interface guarantees that the address is in lowercase